File: Solutions/453. Minimum Moves to Equal Array Elements/453. Minimum Moves to Equal Array Elements.cpp
class Solution {
public:
    int minMoves(vector<int>& nums) {
        long sum=0;
        for(int i=0;i<nums.size();++i)
            sum+=nums[i];
        return sum-*min_element(nums.begin(),nums.end())*nums.size();
    }
};

File: Solutions/453. Minimum Moves to Equal Array Elements/453. Minimum Moves to Equal Array Elements.java
class Solution {
    public int minMoves(int[] nums) {
        int sum=0,m_in=Integer.MAX_VALUE;
        for(int i=0;i<nums.length;++i){
            sum+=nums[i];
            m_in=Math.min(m_in,nums[i]);
        }
        return sum-m_in*nums.length;
    }
}